### PR TITLE
Add custom scripts feature

### PR DESCRIPTION
**Pull request by lorcalhost**
Todo:
- [x] Add documentation
- [x] Check for bugs
- [x] Add more emojis to buttons
- [x] Code cleanup

### CODE REVIEW

File: btb_manager_telegram/utils.py [hunk 1]
```diff
@@ -206,3 +207,22 @@ def update_reminder(self, message):
...
+def get_custom_scripts_keyboard():
+    custom_scripts_path = "./config/custom_scripts.json"
+    keyboard = []
+    custom_script_exist = False
+    message = "No custom script was found inside *BTB\-manager\-telegram*'s `/config/custom_scripts.json` file\."
+
+    if os.path.exists(custom_scripts_path):
+        with open(custom_scripts_path) as f:
+            scripts = json.load(f)
+            if len(scripts) > 0:
```
Comment (DmytroLitvinov) [inline btb_manager_telegram/utils.py:221]: You can omit that additional checks if you are not doing custom logic in `else` block.

File: btb_manager_telegram/handlers.py [hunk 2]
```diff
@@ -232,6 +237,23 @@ def menu(update: Update, _: CallbackContext) -> int:
                 parse_mode="MarkdownV2",
             )
 
+    elif update.message.text == "Execute custom script":
+        re = get_custom_scripts_keyboard()
+        if re[1]:
+            kb = re[0]
```
Comment (DmytroLitvinov) [inline btb_manager_telegram/handlers.py:243]: what does it mean `kb`?

Comment (lorcalhost) [reply btb_manager_telegram/handlers.py:243]: `kb` is the keyboard.

File: btb_manager_telegram/utils.py [hunk 3]
```diff
@@ -61,15 +61,22 @@ def setup_telegram_constants():
         exit(-1)
 
 
-def telegram_text_truncator(m_list) -> List[str]:
-    message = [""]
+def telegram_text_truncator(
```
Comment (lorcalhost) [inline btb_manager_telegram/utils.py:64]: Actually `padding_len` is not needed.  
```python
if (
    len(message[index]) + len(mes) + padding_len
    <= telegram.constants.MAX_MESSAGE_LENGTH
):
```

Should become 
```python
if (
    len(message[index]) + len(mes) + len(padding_chars_tail)
    <= telegram.constants.MAX_MESSAGE_LENGTH
):
```

**Pull request merged**